hs_core/views/__init__.py

```diff
@@ -64,6 +65,33 @@
 logger = logging.getLogger(__name__)
 
 
+class HydroshareWebServices():
+
+    def __init__(self, services_url, api_token, timeout):
+        self.services_url = services_url
+        self.timeout = timeout
+        self.session = requests.Session()
+        self.session.headers.update(
+            {"Authorization": " ".join(("Token", str(api_token)))}
+        )
+
+    def update(self, res_id):
+        rest_url = str(self.services_url) + "/update-services/?res_id=" + str(res_id)
+        try:
+            return self.session.post(rest_url, timeout=self.timeout)
+        except requests.exceptions.RequestException as requests_exception:
+            return requests_exception
+        except ValueError as requests_exception:
+            return requests_exception
```

Comment:
These exceptions should be written to the log for later debug and diagnosis, returning exceptions does not serve any purpose here since caller functions don't use them at all. Also `requests_exception``` variable is not visible in the second ValueError except statement and needs to be corrected. In fact, both exceptions can be consolidated into the same except statement to be logged.